site/public/js/testcase-output.js

```diff
@@ -1,3 +1,4 @@
+/* global toggleDiv, buildCourseUrl, enableKeyToClick, displayAjaxError */
```

Comment:
This should have a line of whitespace under it to distinguish it from the function documentation.